client/my-sites/marketplace/components/progressbar/style.scss

```diff
@@ -0,0 +1,4 @@
+
+.marketplace__loading {
+    margin-top: 0 !important;
```

Comment:
Can we avoid the `!important` with better specificity?

Reply:
We can! I added that before I'd actually found out where the rule was coming from 😅